public/plgapp/plgapp.js

```diff
@@ -94,6 +98,37 @@ var PlgApp = function () {
         }
         return new AppError(status + ' ' + xhr.status + ' ' + error, data);
     };
+
+    var setSessionTimeout = function(timeout_callback, seconds_before_timeout) {
```

Comment:
Identifier 'timeout_callback' is not in camel case.<br>Identifier 'seconds_before_timeout' is not in camel case.